python/ray/_private/worker.py

```diff
@@ -813,6 +814,7 @@ def put_object(
                 objects. If True, then the returned object will not have a
                 valid value. The object must be written to using the
                 ray.experimental.channel API before readers can read.
+            tensor_transport: The tensor transport backend to use.
```

Comment:
Can you list valid values?

Reply:
I think we also need to add checks. Seems we'd only support OBJECT_STORE and NIXL right now?